### PR TITLE
Feat: Borderless inputs and textAlign prop on button

### PR DESCRIPTION
**Pull request by i-pip**
## What kind of change does this PR introduce?

Adds borderless inputs and textAlign prop to button

### CODE REVIEW

File: src/components/Textarea/Textarea.css [hunk 1]
```diff
@@ -5,3 +5,7 @@
 .form-textarea--error {
   @apply border-red-500;
 }
+
+.form-textarea--borderless {
+  @apply border-0;
```
Comment (MildTomato) [inline src/components/Textarea/Textarea.css:10]: @i-pip should perhaps be `border-none` it keeps the same width of the border compared to other inputs.

**Comment by MildTomato** [comment]
@i-pip applying a couple of edits, hope that's ok 👍  😄

**Comment by MildTomato** [comment]
fixed Text Area as well, let me know if that's ok 😄

File: src/components/Textarea/Textarea.css (hunk 1 above)
Comment (i-pip) [reply src/components/Textarea/Textarea.css]: Thanks for the review @MildTomato! Didn't know about `border-none` - very cool! I've pushed the commit, thanks for the other changes too.

**Pull request merged**